playground/src/App.vue

```diff
@@ -70,6 +79,10 @@ const scores = computed(() => {
         </div>
         <div class="column size-30 auto-margin">
             <div class="scores">
+                <div class="readability-score">
+                    <h3>Readability Score</h3>
```

Comment:
Nitpick: capitalization

I really like this way of displaying the readability score!

```suggestion
                    <h3>Readability score</h3>
```

Reply:
I'm always forgetting this one 😨